CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+Contributing
+------------
+
+1. [How to contribute](#how-to-contribute)
+    * [Reporting a bug](#reporting-a-bug)
+    * [Writing a pull request](#writing-a-pull-request)
+2. [Documentation](#documentation)
+
+## How to contribute
+
+If you are reading this you probably want to contribute, so thanks! Maintaining an open-source project is hard work and any help is welcome.
+
+### Reporting a bug
+
+If you find a bug, before reporting it, check if it's not [already reported][issues].  
+
+If you're the first one to report it, please, follow those recommendations: 
+- Write a clear and precise title
+- Write a description which contain a scenario on how to reproduce the bug. Feel free to post a link to [Plunker][plunker]
+- You'll never be too precise so don't hesitate to give more informations: OS, browser, node version...
+
+### Writing a pull request
+
+**Please follow these basic steps to simplify pull request reviews - if you don't you'll probably just be asked to anyway.**
+
+Fork the [hm-admin][hm-admin] project and work on your own fork.  
+You have to work on a branch with a clear name (feature/writing-how-to-contribute for example). Prefixing your branch by 
+"feature/" (for new feature) and "hotfix/" (for issue resolving) is a good practice.
+
+When your work is done:
+ 
+- Rebase your branch on the current master
+- Be sure that all tests pass and your code is lint free. You can run `npm test` to check
+- Squash your commits to one commit and follow those recommendations about [writing good commit message][writing-good-commit-message]
+- Build a pull request via github from your yourFork/yourBranch to the hm-admin/master branch
+ 
```

Comment:
It's also possible to add GitHub PR templates: https://github.com/blog/2111-issue-and-pull-request-templates